spec/system/consumer/caching/shops_caching_spec.rb

```diff
@@ -14,12 +14,13 @@
   }
 
   describe "caching enterprises AMS data" do
-    it "caches data for all enterprises, with the provided options" do
+    before do
       # Trigger lengthy tasks like JS compilation before testing caching:
       visit shops_path
       Rails.cache.clear
```

Comment:
🤔 I wonder if we can't trigger JS compilation directly?

Reply:
Well, my previous PR was triggering the whole JS compilation but that was maybe inefficient. Visiting a certain page ensures that we compile only the assets needed for that page.

By the way, I'm not sure if my assessment of significant runtime increase on my previous PR was right. I've seen plenty of other builds taking that long as well. I may have just been unlucky with the build. I didn't look at the detailed spec run times.